pkg/assets/handlers.go

```diff
@@ -84,6 +85,7 @@ func HTML5ModeHandler(h http.Handler) http.Handler {
 				http.Error(w, "Failed to read index.html", http.StatusInternalServerError)
 				return
 			} else {
+				b = bytes.Replace(b, []byte(`<base href="/">`), []byte(fmt.Sprintf(`<base href="%s">`, contextRoot)), 1)
```

Comment:
do this first in HTML5ModeHandler() and reuse the replaced byte array from within the closure